lib/config/options/index.ts

```diff
@@ -437,7 +437,7 @@ const options: RenovateOptions[] = [
   {
     name: 'optimizeForDisabled',
     description:
-      'Set to `true` to first check for disabling in config before cloning.',
+      'Set to `true` to first check for disabling INSERT WHAT WE CHECK FOR HERE in config before cloning.',
```

Comment:
> Set to `true` to first check for disabling in config before cloning.

This sentence is missing some information about what is disabled in config before cloning.

Reply:
Is there more info in the markdown file?

Reply:
I've added markdown docs for this option now